improver/blending/blend_across_adjacent_points.py

```diff
@@ -114,7 +114,8 @@ def _find_central_point(self, cube):
             Unit(self.parameter_units).convert(
                 self.central_point, cube.coord(self.coord).units))
         constr = iris.Constraint(
-            coord_values={self.coord: central_point})
+            coord_values={self.coord: lambda cell:
+                          cell.point == central_point})
```

Comment:
@cgsandford a question about this comparison. The central point returned by the unit call above is a float; should we force it to be an int for the comparison? I'm always nervous of float equality issues, but this plugin could be used with times in hours in which case forcing things to ints would be bad.

You've thought a lot more about time in IMPROVER than most. Thoughts?

Reply:
So I think we'd usually expect time coordinates to be integer, but because this converts units based on a command line argument we can't rely on that.  In which case I suggest:
```
coord_values = {self.coord: lambda cell: np.isclose(cell.point, central_point)}
```